pkg/apis/monitoring/v1beta1/alertmanager_config_types.go

```diff
@@ -295,6 +295,16 @@ type DiscordConfig struct {
 	// HTTP client configuration.
 	// +optional
 	HTTPConfig *HTTPConfig `json:"httpConfig,omitempty"`
+	// The template of the content's body.
```

Comment:
Hi @junaidk is a bit unrelated to your change but for consistency if you don't mind removing the extra space in the rest of the `type DiscordConfig struct {` and also add the kubebuilder validation, in both api versions. Thanks

Reply:
hi @heliapb, I have updated both api versions to make them consistent.